### DISCUSSION TITLE
녹화기능이 궁금해요.

### DISCUSSION THREAD

**Discussion by sucrosekr** [kyechan99/chzzk-plus#6] [Q&A]
녹화를 짧게 하면, 경로지정하는 창이 뜨는데, 
1분정도 이상 녹화를 하면, 경로지정하는 창이 아예 안뜹니다.
혹시 m3u8 주소를 따오는 방식이 아니라면 경로지정을 처음부터 할 수 있는 방법이 있나요?

**Comment by kyechan99** [comment]
@sucrosekr  안녕하세요~
**경로지정하는 창**이 정확히 어떤 뜻일까요?

**Reply by sucrosekr** [reply]
다운로드 창입니다 ㅎㅎ

**Reply by kyechan99** [reply]
![image](https://github.com/kyechan99/chzzk-plus/assets/14146566/9f1bebf0-55ac-40b4-adae-f2830c4104af)

이런 창을 말씀하시는 걸까요?
만약 그렇다면 제 환경에서는 저런 창이 안뜨다보니 정확한 원인을 당장에는 못찾겠네요.
다만 제가 알기로 기본적으로 브라우저에서 다운로드를 진행할때는 다운로드 경로로 자동 지정해 지는 것으로 알고 있는데요.

Q. Mac 환경이시려나요?
Q2. 혹시 1분 이상 파일은 아예 저장조차 되지 않는 상황일까요?

**Reply by sucrosekr** [reply]
네네 맞아요 매킨도시 윈도우 둘다 그러네요
대략 1분이라고 말씀드렸지만 
사실 좀 길다 싶으면 저장조차 되지 않는게 맞습니다
